content/zh-cn/docs/tutorials/stateful-application/basic-stateful-set.md

```diff
@@ -1823,20 +1865,51 @@ above.
 `OrderedReady` Pod 管理策略是 StatefulSet 的默认选项。它告诉
 StatefulSet 控制器遵循上文展示的顺序性保证。
 
+<!--
+Use this when your application requires or expects that changes, such as rolling out a new
+version of your application, happen in the strict order of the ordinal (pod number) that the StatefulSet provides.
+In other words, if you have Pods `app-0`, `app-1` and `app-2`, Kubernetes will update `app-0` first and check it.
+Once the checks are good, Kubernetes updates `app-1` and finally `app-2`.
+-->
+当你的应用程序需要或期望更改（例如推出应用程序的新版本）按照 StatefulSet
+提供的序号（Pod 编号）的严格顺序发生时，请使用此选项。
+换句话说，如果你有 Pod `app-0`、`app-1` 和 `app-2`，Kubernetes 将首先更新 `app-0` 并检查它。
+一旦检查良好，Kubernetes 就会更新 `app-1`，最后更新 `app-2`。
+
+<!--
+If you added two more Pods, Kubernetes would set up `app-3` and wait for that to become healthy before deploying
+`app-4`.
+
+Because this is the default setting, you've already practised using it.
+-->
+如果你再添加两个 Pod，Kubernetes 将设置 `app-3` 并等待其正常运行，然后再部署 `app-4`。
+
+因为这是默认设置，所以你已经在使用它。
```

Comment:
The nuance to capture here is “&hellip;and by implication, the tutorial will not make you run through the same steps once again”.